### PR TITLE
View products front end

### PR DESCRIPTION
**Pull request by judyichiang**
image

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import Header from './header';
+// import ProductList from './product-list';
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please do not turn in commented out code.

File: client/components/app.jsx [hunk 2]
```diff
@@ -18,8 +20,9 @@ export default class App extends React.Component {
   }
 
   render() {
-    return this.state.isLoading
-      ? <h1>Testing connections...</h1>
-      : <h1>{this.state.message}</h1>;
+    return (
+      <Header name="Wicked Sales"/>
+      // <ProductList />
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please do not turn in commented out code.

File: database/dump.sql [hunk 3]
```diff
@@ -1,137 +0,0 @@
---
```
Comment (thebearingedge) [inline database/dump.sql]: Please re-export your database. This file should be committed.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by judyichiang** [comment]
I really did it this time!
image

File: client/components/product-list-item.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            <div className="card-body">
+              <h5 className="card-title">{product.name}</h5>
+              <p className="card-text">${(product.price / Math.pow(10, 2)).toFixed(2)}</p>
+              <p className="card-text">{product.shortDescription}</p>
+            </div>
+          </div>
+
+        </div>
+
+      ));
```
Comment (thebearingedge) [inline client/components/product-list-item.jsx]: Ok, so the mapping should happen in the list component. Take a look at `fetch-in-react`, `fetch-in-react-practice`, and `sgt-react`. The single-item components are only responsible for showing one thing and the list components are responsible for mapping through the data array.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**